### DISCUSSION TITLE
Parallelized version of LPCMCI

### DISCUSSION THREAD

**Discussion by rcrdz** [jakobrunge/tigramite#278] [Ideas]
Hi tigramite developers!

I am currently writing my master's thesis on different methods of causal discovery.
I would very much like to try the LPCMI with all my variables (>60). Unfortunately, the procedure takes way too long on my local computer. 
I saw that there is a version of the code for parallelizing PCMCI, but unfortunately not for LPCMCI.
Would it be possible to provide a version for running LPCMCI in parallel?

Thanks a lot in advance!

Best,
Moritz

**Comment by jakobrunge** [comment]
This would be a nice project, but we don't have the human resources right now...
